src/clj/harja/kyselyt/toteumat.sql

```diff
@@ -463,14 +463,14 @@ SELECT ut.tehtava               AS toimenpidekoodi_id,
        sum(ut.maara)            AS suunniteltu_maara,
        tk.kasin_lisattava_maara AS kasin_lisattava_maara,
        tk.suunnitteluyksikko    AS yk,
-       'tyyppi'                 AS tyyppi
+       'kokonaishintainen'      AS tyyppi
 FROM urakka_tehtavamaara ut
          JOIN toimenpidekoodi tk ON tk.id = ut.tehtava
          JOIN tehtavaryhma tr ON tr.id = tk.tehtavaryhma AND (:tehtavaryhma::TEXT IS NULL OR tr.otsikko = :tehtavaryhma)
 WHERE ut.urakka = :urakka
   AND ut."hoitokauden-alkuvuosi" = :hoitokauden_alkuvuosi
   -- Rajataan edellisessä haussa löydetyt toteumat, joilla on suunnitelma ja määrä olemassa, pois tästä
```

Comment:
Tänne pitäs käsittääkseni lisätä `ut.poistettu IS NOT TRUE`. Ainakin, jos kannassa merkkaa jonku (esim. tehtävän K2) poistetuksi, niin se siitä huolimatta näkyy frontilla.

Reply:
Kiitos. Tarkistus puuttui myös tuolta vähän ylempää.